js/script.js

```diff
@@ -1,16 +1,18 @@
 // Book Class: Represents a Book
 class Book {
-  constructor(title, author, isbn) {
+  constructor(title, author, email, phone, isbn) {
```

Comment:
Doing this change will effect the existing code instantiating the Book class. Consider using constructor overloading, e.g. constructor(title, author, isbn, email = "", phone = "")